src/lib.rs

```diff
@@ -505,11 +560,91 @@ impl Navigator {
         todo!()
     }
 
-    pub fn set_led_on(&mut self) {
+    /// Sets the navigator LED output based on it's color.
+    ///
+    /// # Arguments
+    ///
+    /// * `pin` - A pin selected by LED's color from [`LedColor`](enum.LedColor.html).
+    /// * `state` - The value of output, LED is on with a true logic value.
+    ///
+    /// # Examples
+    ///
+    /// ```no_run
+    /// use navigator_rs::{LedColor, Navigator};
+    /// use std::thread::sleep;
+    /// use std::time::Duration;
+    ///
+    /// let mut nav = Navigator::new();
+    ///
+    /// nav.init();
+    /// loop {
+    ///     nav.set_led(LedColor::Green, true);
+    ///     sleep(Duration::from_millis(1000));
+    ///     nav.set_led(LedColor::Green, false);
+    ///     sleep(Duration::from_millis(1000));
+    ///     nav.set_led(LedColor::Blue, true);
+    ///     sleep(Duration::from_millis(1000));
+    ///     nav.set_led(LedColor::Blue, false);
+    ///     sleep(Duration::from_millis(1000));
+    ///     nav.set_led(LedColor::Red, true);
+    ///     sleep(Duration::from_millis(1000));
+    ///     nav.set_led(LedColor::Red, false);
+    ///     sleep(Duration::from_millis(1000));
```

Comment:
that appears really unnecessary.. please make the examples simpler.

Reply:
it was blinking pretty well, but okay. will keep just green led.